packages/react-experiments/src/components/SelectedItemsList/Items/subcomponents/DefaultEditingItem.tsx

```diff
@@ -41,118 +45,185 @@ export interface IDefaultEditingItemInnerProps<TItem> extends React.HTMLAttribut
    * Callback used by the EditingItem to populate the initial value of the editing item
    */
   getEditingItemText: (item: TItem) => string;
+
+  /**
+   * Callback used to retrieve the suggestions to show in the floating suggestions
+   */
+  getSuggestions: (value: string) => IFloatingSuggestionItemProps<TItem>[];
```

Comment:
In the old picker, the equivalent to this function returned either the items or a promise. Not sure if I need the promise here as well?

Reply:
Can't remember but that might be because of waiting on an async call to the server to get the suggestion list?

Reply:
I think so. Looks like that's the "older" implementation so going to leave it out for now, may need it later we'll see